common-ts/src/types/UIEnv.ts

```diff
@@ -0,0 +1,58 @@
+import { DriftEnv } from '@drift-labs/sdk';
+import { Opaque } from '.';
+
+// Define UIEnvKey as an opaque type
+export type UIEnvKey = Opaque<'UIEnvKey', string>;
+
+/**
+ * Utility class to handle environment configs in the Drift UI. Note that these environments don't exactly align with the SDK environment (SDK can only be mainnet or devnet, but UI can be devnet, staging, mainnet) so there is a `sdkEnv` getter utility to convert the UI env to SDK env (UI staging => SDK mainnet).
+ */
+export class UIEnv {
+	constructor(readonly env: DriftEnv | 'staging') {}
+
+	static createMainnet() {
+		return new UIEnv('mainnet' as DriftEnv);
```

Comment:
why not mainnet-beta?

Reply:
mainly for consistency with this : https://github.com/drift-labs/drift-common/blob/master/common-ts/src/EnvironmentConstants.ts